app/assets/javascripts/maps.js

```diff
@@ -169,7 +173,7 @@ function setMarkers(){
   // Add the markers and infowindows to the map
   var iconCounter = 0;
   for (var i = 0; i < markers.length; i++) {  
-    var description = markers[i][1];
+    var description = markers[i][4];
```

Comment:
This is the only place `markers[i][1]` is being used. You can probably change the controller to just pass in a link instead of the name."